src/js/pages/Setup.js

```diff
@@ -464,12 +513,49 @@ class Setup extends mixin(StoreMixin) {
           validationErrorText: this.getErrors('ip_detect_script'),
           value: this.state.formData.ip_detect_script
         }
-      ]
+      ],
+      {
+        fieldType: 'checkboxMultiple',
+        name: 'nested_cluster_options',
+        formRowType: 'inline',
+        value: [
+          {
+            name: 'telemetry_enabled',
+            label: this.getCheckboxLabel('Send Anonymous Telemetry', (<span>
+                Enter or upload a script that runs on each node in the
+                cluster and outputs the node’s local IP address. <a
+                  href={`${Config.documentationURI}/advanced-installer/create-a-script-for-ip-address-discovery/`}
+                  target="_blank">
+                  Learn more
+                </a>.
+              </span>)),
+            checked: this.state.formData.telemetry_enabled
+          },
+          {
+            name: 'oauth_enabled',
+            label: this.getCheckboxLabel('Enable Authentication', (<span>
+                Enter or upload a script that runs on each node in the
+                cluster and outputs the node’s local IP address. <a
+                  href={`${Config.documentationURI}/advanced-installer/create-a-script-for-ip-address-discovery/`}
+                  target="_blank">
+                  Learn more
+                </a>.
+              </span>)),
+            checked: this.state.formData.oauth_enabled
+          }
+        ]
+      }
     ];
 
     return Hooks.applyFilter('FormDefinition', formDefintion);
   }
 
+  getFormRowClass(definition) {
```

Comment:
It needs to be a method because that's the only way to assign a different class name for different rows in the `Form` component. Here's a related discussion: https://github.com/dcos/dcos-installer-ui/pull/2/files/1db0baf52afa06310fb92845527d697476734492#diff-67625111cd35b2a45d7fa0eb6f1236d8R516